src/main/java/com/redhat/cloud/notifications/db/ApplicationResources.java

```diff
@@ -139,8 +139,7 @@
                     .setFirstResult(limiter.getLimit().getOffset());
         }
 
-        return mutinyQuery.getResultList()
-                .onItem().transformToMulti(Multi.createFrom()::iterable);
+        return mutinyQuery.getResultList();
```

Comment:
What's the preference for returning a `Multi` vs a `Uni<List>` ? Is this because of sql session?

Reply:
`Uni` is a simpler and lightweight version of `Multi`. We should always favor `Uni` over `Multi` when we don't need any of the `Multi` features, which is the case in all modified classes in this PR. Hibernate always returns `Uni` for all of its asynchronous operations, which is one more reason to stick with `Uni` whenever possible. There are of course many cases when we need a `Multi` and it's perfectly fine to use them, even though some things we do with `Multi` could be done with simple Java streams in notifications-backend.